db/c_test.c

```diff
@@ -126,6 +126,32 @@ static void CheckGetCF(
   Free(&val);
 }
 
+static void CheckPinGet(rocksdb_t* db, const rocksdb_readoptions_t* options,
+                        const char* key, const char* expected) {
+  char* err = NULL;
+  size_t val_len;
+  const char* val;
+  rocksdb_pinnableslice_t* p;
```

Comment:
Why does it have to be created on heap? Can't it be simply an on-stack object that is passed to rocksdb_get_pinned?